src/aer-types/types/data.ts

```diff
@@ -90,6 +90,12 @@ export type Expansions = {
   [id: string]: Expansion
 }
 
+export function formatExpansionName(expansion: Expansion): string {
+  return expansion.wave
+    ? expansion.name + ' (' + expansion.wave + ')'
```

Comment:
Could you please use a template literal instead of the manual concatenation here:

```ts
`${expansion.name} (${expansion.wave})`
```